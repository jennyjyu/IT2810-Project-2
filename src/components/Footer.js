import React from 'react';


export default function Footer(props)  {
    return (
        <div className="footer-container">
            <h3>
                {props.footerText}

            </h3>
        </div>
    )
}
