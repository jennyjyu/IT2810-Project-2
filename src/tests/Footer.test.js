import React from "react"
import Footer from "../components/Footer"
import renderer from "react-test-renderer"
// import App from "../App"



test("Footer snapshot test", () => {
  const component = renderer.create(
    <Footer/>
  )
  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
