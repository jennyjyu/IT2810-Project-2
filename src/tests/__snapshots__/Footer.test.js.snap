// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot test 1`] = `
<div
  className="footer-container"
>
  <h3 />
</div>
`;
